CHANGES.txt

```diff
@@ -8,6 +8,10 @@ NOTE: The 4.0.0 Release of SCons dropped Python 2.7 Support
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
+  From Joseph Brill:
```

Comment:
Does this have any user impact? If so be specific about what this is fixing, if not then say internal logic only. Just so it's easier for someone not super familiar with SCons to understand the impact.

Reply:
Added "Internal".